libraries/networking/src/udt/Connection.h

```diff
@@ -73,10 +73,12 @@ class Connection : public QObject {
 
 signals:
     void packetSent();
+    void connectionInnactive(HifiSockAddr sockAdrr);
```

Comment:
Okay - looked this up. If direct, you get the same instance as a `const HifiSockAddr&`. If queued, they'll copy it.

I say this means we should use a const ref on the off chance that a connection is direct.